styles/style.css

```diff
@@ -149,26 +188,23 @@ nav a:hover {
 .hero {
     background: linear-gradient(0deg, rgba(0 0 0 / 50%), rgba(0 0 0 / 50%)), url(/images/iceland_1920_compress.jpg);
     background-position: center;
-    background-size: 165vh;
-    padding: 12px;    
-    height: 50vh;
-    display: flex; 
+    background-size: cover;    
+    display: flex;     
```

Comment:
Well done on rearranging the CSS properties. This improves the readability of your css file